extern class EvalError

export func foo() ~ EvalError {

}